abstract.go

```diff
@@ -16,6 +16,16 @@ var (
 	tagSemaphore        chan struct{}
 )
 
+func getRegions(region string, regions []string) []string {
```

Comment:
Whole logic is no more needed as far as i can see, if we remove region.

Reply:
same as above, i'll update it.